stdlib/stdlib.fc

```diff
@@ -645,3 +645,22 @@ builder store_builder(builder to, builder from) asm "STBR";
 
 ;;; Retrieves code of smart-contract from c7
 cell my_code() asm "MYCODE";
+
```

Comment:
Maybe, it would be handy to visually separate the regular `stdlib.fc` (from TON monorepo) and the additions you make specifically for Tact's use. Like, `;;; --- Tact-adjusted ---`, or something

Reply:
well, there's nothing adjusted specifically for Tact. I took the "newer" stdlib from one of the recent core projects and removed the stuff that they put there for their FunC code (helper functions for building cells and some constants already existing in Tact etc)